src/main/java/frc/robot/RobotContainer.java

```diff
@@ -215,8 +215,8 @@ private void setupDriver() {
         Command autoScore = CommandFactory
             .autoScore(swerve, elevator, coralScoring, algae, operator::getDesiredLocation,
                 operator::getDesiredHeight, operator::additionalAlgaeHeight, operator::crossOut)
-            // .andThen(CommandFactory.doSomethingWithAlgae(swerve, elevator, intakingAlgae, algae,
-            // operator::whatToDoWithAlgae, () -> -driver.getLeftX()))
+            .andThen(CommandFactory.doSomethingWithAlgae(() -> true, swerve, elevator, algae,
```

Comment:
we need to update this to use an actual supplier of having algae and if we should do something with it.